src/game/savegame/savegame_bson.c

```diff
@@ -547,13 +547,6 @@ static bool Savegame_BSON_LoadItems(struct json_array_s *items_arr)
             } else if (obj->intelligent) {
                 item->data = NULL;
             }
-
-            if (item->object_number == O_FLAME_EMITTER
-                && g_Config.enable_enhanced_saves) {
-                int32_t flame_num =
-                    json_object_get_int(item_obj, "flame_num", 0);
-                item->data = (void *)flame_num;
-            }
```

Comment:
@rr- will I need a version update for this? I think taking out this load check shouldn't cause issues except for old saves won't restore the flame emitter properly. But that will return to OG behavior which I think is better than risking a crash even if it was rare.

Reply:
No it's OK. I'd mention this in the changelog though.